sky/setup_files/dependencies.py

```diff
@@ -140,6 +131,7 @@
         # docs instead.
         # 'vsphere-automation-sdk @ git+https://github.com/vmware/vsphere-automation-sdk-python.git@v8.0.1.0' pylint: disable=line-too-long
     ],
+    'nebius': ['nebius>=0.2.0',]
 }
 
 extras_require['all'] = sum(extras_require.values(), [])
```

Comment:
Can we exclude nebius from `all` when running on python 3.9? Else py3.9 users won't be able to run `pip install skypilot[all]`. 

Wondering if a conditional filter like this could work:
```
if sys.version_info < (3, 10):
    # Exclude 'nebius' from the keys when building 'all'
    all_keys = [k for k in extras_require.keys() if k != 'nebius']
    extras_require['all'] = sum((extras_require[k] for k in all_keys), [])
else:
    extras_require['all'] = sum(extras_require.values(), [])
```

Reply:
We should probably leave a comment here about why we put this filter.